src/Microsoft.ServiceFabric.Actors/Runtime/ActorConcurrencyLock.cs

```diff
@@ -200,39 +200,39 @@ internal int Test_CurrentCount
             }
             catch
             {
-                // dirty handler threw and exception, release the turn lock
-                // and throw the exception back
+                // Dirty handler threw and exception, release the turn lock
+                // and throw the exception back.
                 this.turnLock.Release();
                 throw;
             }
 
-            // release the reentrancy lock but continue to hold the turn lock and release 
-            // it through ReleaseContext method after this call invocation on the actor is completed
+            // Releases the reentrancy lock but continue to hold the turn lock and release 
+            // it through ReleaseContext method after this call invocation on the actor is completed.
 
-            // indicate that the turn based concurrency lock is acquired and proceed to 
-            // call the method on the actor
+            // Indicates that the turn based concurrency lock is acquired and proceed to 
+            // call the method on the actor.
 
         }
 
         public async Task ReleaseContext(string callContext)
         {
-            // first acquire the reentrancy lock to reduce the call counts and 
+            // First acquire the reentrancy lock to reduce the call counts.
             await this.reentrantLock.WaitAsync();
             try
             {
-                // this call must be part of the original call chain
+                // This call must be part of the original call chain.
                 if (callContext.StartsWith(this.currentCallContext))
                 {
                     if (this.currentCallCount > 0)
                     {
-                        // reduce the current call count as this call is finishing
+                        // Reduces the current call count as this call is finishing.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.